other/fun/CMakeLists.txt

```diff
@@ -6,36 +6,42 @@ function(target_link_toxcore target)
   endif()
 endfunction()
 
+function(target_link_sodium target)
```

Comment:
@iphydf did we not have a better solution for this using alias/imported/fake target?

Reply:
I am ok with this when you copy the full portion from here and not only a part: https://github.com/TokTok/c-toxcore/blob/ed2b60c2177808374a29c22effd30414ec0bffae/testing/CMakeLists.txt#L11-L18

Reply:
Done